sdk/containerregistry/Azure.Containers.ContainerRegistry/src/Models/ArtifactManifestProperties.cs

```diff
@@ -17,13 +17,13 @@ public ArtifactManifestProperties()
         {
         }
 
-        /// <summary> Delete enabled. </summary>
+        /// <summary> Whether or not this tag can be deleted. </summary>
```

Comment:
Does it make sense to add the context like we did for `CanList`?
```suggestion
        /// <summary> Whether or not this tag can be deleted with <see cref="DeleteTag" /> . </summary>
```

Reply:
There's non-trivial usage of the Azure CLI for this service, so I'd like to leave it general to that.